sale_order_invoicing_queued/models/queue_job.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2019 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo import models
+
+
+class QueueJob(models.Model):
+    _inherit = 'queue.job'
+
+    def cancel_sudo(self):
+        self.sudo().filtered(
+            lambda x: x.state in ['pending', 'enqueued']
+        ).unlink()
+
+    def requeue_sudo(self):
+        self.sudo().requeue()
```

Comment:
For the same reason, use ACLs here too, instead of sudo.

Reply:
I remove the method at all, as no sudo is required.